numpy-stubs/__init__.pyi

```diff
@@ -215,8 +208,7 @@ class flatiter:
 
 
 _ArraySelf = TypeVar("_ArraySelf", bound=_ArrayOrScalarCommon)
-class _ArrayOrScalarCommon(SupportsInt, SupportsFloat, SupportsComplex,
```

Comment:
Did something change to explain why there's no longer a point in subclassing from `SupportsInt`, etc?

Reply:
I think protocols make this subclassing unnecessary.